main.py

```diff
@@ -50,19 +50,19 @@ def boot():
     logger.step("Boot")
     logger.substep("Opening terminal and accept user input")
     logger.info("+----------------------------------------+")
-    logger.info("|             DamienOS build             |")
+    logger.info("|             AlphaOS build             |")
```

Comment:
OMG THE THING IS OFF.  ONE CHARACTER OFF.

Reply:
It doesn't matter...